ui/.eslintrc.js

```diff
@@ -69,7 +70,7 @@ module.exports = {
                 group: "internal",
               },
             ],
-            "pathGroupsExcludedImportTypes": ["react"],
+            pathGroupsExcludedImportTypes: ["react"],
             "newlines-between": "always",
```

Comment:
well, quite right, this isn't json, but should the next line be fixed too? `newlines-between`?

Reply:
That was just prettier doing its thing. For `newlines-between` the key contains a hyphen so has to have quotes to be valid.